net/FlatBuffers/ByteBuffer.cs

```diff
@@ -70,13 +78,7 @@ public int Length
             protected set;
         }
 
-        public abstract void Dispose();
-
         public abstract void GrowFront(int newSize);
-
-#if !ENABLE_SPAN_T
-        public abstract byte[] ByteArray { get; }
-#endif
     }
 
     public class ByteArrayAllocator : ByteBufferAllocator
```

Comment:
This should be sealed unless we expect folks to derive from it.  That'll allow for devirtualization in some cases by the JIT.

Reply:
Will add.